person.rb

```diff
@@ -0,0 +1,36 @@
+require './nameable'
+
+class Person < Nameable
+  def initialize(age, rentals, name = 'Unknown', _parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @_parent_permission = parent_permission
+    @rentals = rentals
+    @visits = []
+  end
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
```

Comment:
- [ OPTIONAL ] Also rental should be put in attr_accessor since we need it as getter an setter